website/docs/tutorials/01-introduction.md

```diff
@@ -4,39 +4,31 @@ sidebar_label: Introduction
 
 # Introduction
 
-Protostar is a StarkNet smart contract development [toolchain](https://en.wikipedia.org/wiki/Toolchain), which helps you with the following tasks:
-
-- [dependencies management](06-dependencies-management.md)
-- [project compilation](05-compiling.md)
-- [testing contracts](07-testing/README.md)
-
-:::note
-Protostar takes its inspiration from [Foundry](https://github.com/foundry-rs/foundry).
-:::
+Protostar is a toolchain for developing StarkNet smart contracts that helps with tasks such as [dependencies management](06-dependencies-management.md), [project compilation](05-compiling.md), and [testing contracts](07-testing/README.md).
+It is inspired by [Foundry](https://github.com/foundry-rs/foundry).
 
 ## Target audience
-
-This guides assume you have basic knowledge about [Cairo and StarkNet](https://www.cairo-lang.org/docs/) and Git.
+This guide assumes that you have basic knowledge of [Cairo, StarkNet](https://www.cairo-lang.org/docs/), and Git.
```

Comment:
and was better

Reply:
IMO it was worse.
- There was a grammar error (it should be "these guides", or "this guide" not "this guides").
- "And" was used twice in a list of three items.

Reply:
Ah you are right, git is there, then lets change it